cypress/support/pages/profile.pageObject.js

```diff
@@ -0,0 +1,16 @@
+import PageObject from '../PageObject';
+
+class profilePageObject extends PageObject {
+
+usernameValue(name) {
+  cy.getByDataCy('profile-username')
+    .should('contain', name)
+}
+
+bioValue(bio) {
```

Comment:
Function names should be in camelCase and should be more descriptive. Rename 'bioValue' to 'checkBioValue'

Reply:
done